### PR TITLE
Portofolio Project

### PR DESCRIPTION
**Pull request by TiteLunda**
This project is about my personal portfolio

### CODE REVIEW

**Review by MeqdamAlqudah — COMMENTED** [review]
Hi @TiteLunda 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by TiteLunda** [comment]
@MeqdamAlqudah Could you please check well? My side i am able to see and read it. kindly check

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <a href="" id="logo">
+            Welcome
+        </a>
+        <nav class="menu">
+            <i class="fa-solid fa-bars "></i>
+        </nav>
+    </header>
+    <section class="header">
+        <div class="container">
+            <h1 class="identity">
```
Comment (iambenkis) [inline index.html]: - [ ] Good job creating your first project at microverse  😄 , And fortunately it looks good 👏 ,  however your page does not have a background-image ,  make sure the background image you use that is used in the Figma 👍 

|  Your Design |  Figma Design |
| :---: | :---: |
| Capture 2 |  Capture|

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<body>
+    <header class="navbar container fixed-top">
+        <a href="" id="logo">
+            Welcome
+        </a>
+        <nav class="menu">
+            <i class="fa-solid fa-bars "></i>
+        </nav>
+    </header>
+    <section class="header">
```
Comment (iambenkis) [inline index.html]: - [ ] It could be better if you use a class name that is not a key word in `html` 👍  , kindly change that 😄

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @TiteLunda 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**